diff -Naurb rspamd-3.6.orig/src/libserver/ssl_util.c rspamd-3.6/src/libserver/ssl_util.c
--- rspamd-3.6.orig/src/libserver/ssl_util.c	2023-08-03 19:26:19.000000000 +0530
+++ rspamd-3.6/src/libserver/ssl_util.c	2023-08-08 08:35:13.360527084 +0530
@@ -1095,7 +1095,7 @@
 void rspamd_ssl_ctx_config(struct rspamd_config *cfg, gpointer ssl_ctx)
 {
 	struct rspamd_ssl_ctx *ctx = (struct rspamd_ssl_ctx *) ssl_ctx;
-	static const char default_secure_ciphers[] = "HIGH:!aNULL:!kRSA:!PSK:!SRP:!MD5:!RC4";
+	static const char default_secure_ciphers[] = "PROFILE=SYSTEM";
 
 	if (cfg->ssl_ca_path) {
 		if (SSL_CTX_load_verify_locations(ctx->s, cfg->ssl_ca_path,
