--- rspamd-3.7.5/src/libserver/ssl_util.c.orig	2023-12-05 07:49:47.680818537 +0530
+++ rspamd-3.7.5/src/libserver/ssl_util.c	2023-12-05 07:50:26.251380202 +0530
@@ -1095,7 +1095,7 @@
 void rspamd_ssl_ctx_config(struct rspamd_config *cfg, gpointer ssl_ctx)
 {
 	struct rspamd_ssl_ctx *ctx = (struct rspamd_ssl_ctx *) ssl_ctx;
-	static const char default_secure_ciphers[] = "HIGH:!aNULL:!kRSA:!PSK:!SRP:!MD5:!RC4";
+	static const char default_secure_ciphers[] = "PROFILE=SYSTEM";
 
 	if (cfg->ssl_ca_path) {
 		if (SSL_CTX_load_verify_locations(ctx->s, cfg->ssl_ca_path,
