diff --git a/src/libserver/ssl_util.c b/src/libserver/ssl_util.c
index 995d46cc6..a75fd888d 100644
--- a/src/libserver/ssl_util.c
+++ b/src/libserver/ssl_util.c
@@ -1076,7 +1076,7 @@ void
 rspamd_ssl_ctx_config (struct rspamd_config *cfg, gpointer ssl_ctx)
 {
 	struct rspamd_ssl_ctx *ctx = (struct rspamd_ssl_ctx *)ssl_ctx;
-	static const char default_secure_ciphers[] = "HIGH:!aNULL:!kRSA:!PSK:!SRP:!MD5:!RC4";
+	static const char default_secure_ciphers[] = "PROFILE=SYSTEM";
 
 	if (cfg->ssl_ca_path) {
 		if (SSL_CTX_load_verify_locations (ctx->s, cfg->ssl_ca_path,
@@ -1111,4 +1111,4 @@ rspamd_ssl_ctx_free (gpointer ssl_ctx)
 
 	rspamd_lru_hash_destroy (ctx->sessions);
 	SSL_CTX_free (ctx->s);
-}
\ No newline at end of file
+}
